modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/block/BlockStorage.scala

```diff
@@ -316,97 +313,104 @@ object BlockStorage {
     val errorMessage: String
     override def getMessage: String = errorMessage
   }
-  case class TipUsageUpdateError[B <: Block[_]](
+
+  case class TipUsageUpdateError(
     child: ProofsHash,
     parent: ProofsHash,
-    encountered: Option[StoredBlock[B]]
+    encountered: Option[StoredBlock]
   )(
-    implicit s: Show[StoredBlock[B]]
+    implicit s: Show[StoredBlock]
   ) extends BlockStorageError {
 
     val errorMessage: String =
       s"Parent block with hash=${parent.show} not found in majority when updating usage! Child hash=${child.show}. Encountered state: ${encountered.show}"
   }
-  case class BlockAcceptanceError[B <: Block[_]](hash: ProofsHash, encountered: Option[StoredBlock[B]])(
-    implicit s: Show[StoredBlock[B]]
+
+  case class BlockAcceptanceError(hash: ProofsHash, encountered: Option[StoredBlock])(
+    implicit s: Show[StoredBlock]
```

Comment:
Not sure you need this implicit here now.

Reply:
does not work without this implicit